src/components/Typography.js

```diff
@@ -41,8 +41,9 @@ export const I = styled.i`
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  border-bottom: 1px solid ${p => p.theme.colors.link};
-  color: ${p => p.theme.colors.link};
+  font-weight: ${p => (p.weight ? 600 : 400)};
```

Comment:
I think this property should be named `bolded` to represent how it styles the text
```suggestion
  font-weight: ${p => (p.bolded ? 600 : 400)};
```